docs/pages/enterprise/sso/adfs.mdx

```diff
@@ -1,16 +1,14 @@
 ---
 title: SSH Authentication With ADFS How To
 description: How to configure SSH access with Active Directory (ADFS) using Teleport
-h1: SSH Authentication with ADFS
+h1: Active Directory as an SSO provider for SSH Authentication
```

Comment:
Technically the SSH authentication is via certificates. Active Directory is providing Teleport Authentication, not SSH authentication.